dbms/src/Storages/DeltaMerge/Delta/ColumnFilePersistedSet.h

```diff
@@ -69,19 +69,19 @@ class ColumnFilePersistedSet : public std::enable_shared_from_this<ColumnFilePer
     size_t next_compaction_level = 0;
     UInt64 minor_compaction_version = 0;
 
-    Poco::Logger * log;
+    LoggerPtr log;
 
 private:
     inline void updateColumnFileStats();
 
     void checkColumnFiles(const ColumnFilePersistedLevels & new_column_file_levels);
 
 public:
-    explicit ColumnFilePersistedSet(PageId metadata_id_, const ColumnFilePersisteds & persisted_column_files = {});
+    explicit ColumnFilePersistedSet(const std::string & log_prefix_, PageId metadata_id_, const ColumnFilePersisteds & persisted_column_files = {});
```

Comment:
use `String`?

Reply:
Not sure about the idiom, but seems that in our code base both `std::string` and `String` are heavily used :/